api/services/ocean/rightsizing/schemas/oceanRightsizingRuleMonthlyRepetitionBasis.yaml

```diff
@@ -0,0 +1,19 @@
+type: object
+title: Ocean Rightsizing Rule Monthly Repetition Basis
+description: >
+  Determines the Ocean Rightsizing rule monthly repetition basis
+properties:
+  intervalMonths:
+    type: array
+    description: |
+      Array of the months (in number), when we want to trigger the apply recommendations.
+    items:
+      type: integer
+      example: 2
+  weekOfTheMonth:
```

Comment:
weekOfTheMonth should be an array of strings (Enums) not a single string. Please recheck.